nuxt.config.cuahsi.js

```diff
@@ -0,0 +1,333 @@
+      // { src: '/vendor/js/gtm.js', type: 'text/javascript' },
+      // {
+      //   src: `https://www.googletagmanager.com/gtm.js?id=${googleTrackingId}`,
+      //   type: 'text/javascript',
+      //   async: 'async'
+      // }
+    ]
+  },
+
+  ignore: widgetsOnly
+    ? [
+        '/pages/*.vue',
+        '/pages/companies/**',
+        '/pages/equipment/**',
+        '/pages/orgs/**'
+      ]
+    : ['/pages/widgets/**'],
+
+  /*
+   ** Customize the progress-bar color
+   */
+  loading: { color: '#fff' },
+
+  /*
+   ** Nuxt.js modules
+   */
+  modules: [
+    // Doc: https: //github.com/robcresswell/nuxt-compress
+    'nuxt-compress'
+  ],
+
+  /*
+   ** Plugins to load before mounting the App
+   ** https://nuxtjs.org/guide/plugins
+   */
+  plugins: [
+    { src: '~/plugins/logger', ssr: false },
+    { src: '~/plugins/tracker', ssr: false },
+    { src: '~/plugins/route', ssr: false },
+    { src: '~/plugins/vee-validate', ssr: false },
+    { src: '~/plugins/ability', ssr: false },
+    { src: '~/plugins/api', ssr: false },
+    { src: '~/plugins/bus', ssr: false },
+    { src: '~/plugins/date', ssr: false },
+    { src: '~/plugins/highcharts', ssr: false },
+    { src: '~/plugins/maps', ssr: false },
+    { src: '~/plugins/global-filters', ssr: false },
+    { src: '~/plugins/global-mixin', ssr: false },
+    { src: '~/plugins/web-workers', ssr: false }
+  ],
+
+  /*
+   ** Nuxt routes
+   ** See https://nuxtjs.org/api//configuration-router
+   */
+  router: {
+    middleware: ['auth', 'ability'],
+    extendRoutes(routes, resolve) {
+      if (presentation === 'dendra') return
+
+      // Find the Dendra default page for each alternate presentation page
+      // example: "-about-cuahsi.vue" -> "about.vue". The "-" ensures that routes are hidden during default presentation
+      const presentationRoutes = glob
+        .sync(`src/pages/-*${presentation}.vue`)
+        .map(fullPath => {
+          return fullPath
+            .replace('src/pages/-', '/')
+            .replace(`-${presentation}.vue`, '')
+        })
+
+      // Create Nuxt alias for each of the alternate presentations
+      routes = routes.map(route => {
+        if (presentationRoutes.includes(route.path)) {
+          route.alias = `/-${route.path.substring(1)}-${presentation}`
+          route.component = resolve(__dirname, `src/pages${route.alias}.vue`)
+        }
+        // index has to be handled separately
+        if (route.path === '/' && presentationRoutes.includes('/index')) {
+          route.alias = `/-index-${presentation}`
+          route.component = resolve(__dirname, `src/pages${route.alias}.vue`)
+        }
+        return route
+      })
+    }
+  },
+
+  /*
+   ** Nuxt application source directory
+   ** See https://nuxtjs.org/api/configuration-dir
+   */
+  dir: {
+    // TODO:DRC consider a better way of asset/static management
+    // assets: 'assets/assets-cuahsi',
+    static: 'static/static-cuahsi'
```

Comment:
@jscottsf this is one downfall of this approach -- asset/static management in this way (re-mapping to `static-cuahsi`) requires a bit of duplication of files within `/static`

Reply:
and in order for `nuxt generate` to work for static build, I think we will need to modify the nuxt config a bit? Not sure, I'll have to look into this more
https://nuxtjs.org/docs/configuration-glossary/configuration-generate/#routes